rfd/0196-opinionated-intro.md

```diff
@@ -0,0 +1,150 @@
+
+Below the context is a list of guides split into themes that form a knowledge map. Individual items can be hidden.
+
+Above the guide is a progress bar or other indicator that gives the user some sense of how close to “done” they are with learning the product. This is not an indicator of them being done fully implementing the product.
+
+Within the content we want to weave a concepts in:
+* Tokens
+* Agents
+* Proxy
+* The role of roles (sorry)
+
+Additionally we want to make it possible for the users to self-select relevant content where possible. To add every implementation form for every topic (i.e. for SSH showing scripted setup, IaC setup, point-and-click setup) would be difficult to maintain and confusing. The point is to be opinionated. By showing 1-2 of our implementation details to each guide users, even if just by reading some, can get a sense of the whole and start to figure out how they want to do things for real. To help users self-select and navigate to relevant content, we’ll tag each guide with methods covered. Within the content we can make nods to other methods (i.e. “you could use these installation steps in a cloudinit script in IaC”).
+
+Tags:
+* UI
+  * Connect
+  * CLI
+  * Console
+* Technology
+  * Linux scripting
+  * Infrastructure-as-Code
+  * Kubernetes CRDs
+  * Point-and-Click
+* Team relevance
+  * Security
+  * Compliance
+  * FIPS/FedRAMP
+  * Scalability
+
+### Rough outline of the content
+* Quickstart
+  * You are a local user, what does that mean?
+  * You have the “access” role, what does that mean?
+  * Add a resource of your choice
+    * Weave in intro to tokens and agents
+  * Access it
+  * Check audit logs
+* Users/Auth
+  * Local
+  * SSO
+* Roles
+  * Strategy
+  * Creation
+  * Assignment
+* Resources
+  * SSH
+  * DB
+    * RDS
+    * Self-Hosted
+  * Application
+  * Cloud Console
+  * Kubernetes
+  * Windows
+  * Machine ID
+* Features
+  * Audit Log/Session Recording
+  * Access Requests
+  * Session Locks/Monitoring
+  * Access Lists
+
+The general flow here is “add users, make roles and assign them, add resources that those users with those roles can now access, do things with the access that has happened.” The quickstart section takes them through that flow for their own user and default role, and then they can expand that schema to the rest of the product.
+
+Key context to provide in each guide is some pros/cons and the why of each topic, especially for topics like Users/Auth. Why use local users vs SSO? This is a good place to incorporate info from SE like in the SSO one saying “you may not have permission to set up your SSO yet, in that case local users are a great place to get started.”
+
+The following fat marker sketches are meant as a jumping off point for design.
+* [side bar and intro page](./assets/0196-sketch1.jpeg)
```

Comment:
We might want to consider how to host / privacy of these videos.  YouTube is often banned, but we could self-host or leverage a 3rd party tool such as https://www.mux.com/ to have greater control.

Reply:
Good call out, and if it adds a lot to scope this is a place I'm fine cutting, just not having the video.